packages/rspack/src/plugins/chunk.ts

```diff
@@ -3,26 +3,26 @@ import webpack from '@rspack/core'
 
 const pluginName = 'ChunkErrorPlugin'
 
-const script = `
-if (typeof ${webpack.RuntimeGlobals.require} !== "undefined") {
-  var _ensureChunk = ${webpack.RuntimeGlobals.ensureChunk};
-  ${webpack.RuntimeGlobals.ensureChunk} = function (chunkId) {
-    return Promise.resolve(_ensureChunk(chunkId)).catch(err => {
-      const e = new Event("nuxt.preloadError");
-      e.payload = err;
-      window.dispatchEvent(e);
-      throw err;
-    });
-  };
-};`
+// const script = `
+// if (typeof ${webpack.RuntimeGlobals.require} !== "undefined") {
+//   var _ensureChunk = ${webpack.RuntimeGlobals.ensureChunk};
+//   ${webpack.RuntimeGlobals.ensureChunk} = function (chunkId) {
+//     return Promise.resolve(_ensureChunk(chunkId)).catch(err => {
+//       const e = new Event("nuxt.preloadError");
+//       e.payload = err;
+//       window.dispatchEvent(e);
+//       throw err;
+//     });
+//   };
+// };`
 
 export class ChunkErrorPlugin {
   apply (compiler: Compiler) {
     compiler.hooks.thisCompilation.tap(pluginName, compilation =>
-      compilation.mainTemplate.hooks.localVars.tap(
-        { name: pluginName, stage: 1 },
-        source => source + script
-      )
+      // compilation.mainTemplate.hooks.localVars.tap(
```

Comment:
Look like `compilation.mainTemplate.hooks.localVars` is update `webpack.RuntimeGlobals.ensureChunk` runtime module, but the implement may inject many `script` logic code. Look like use `runtime_requirements_in_tree` hooks to add the update runtime module.